src/components/landing/EventCard/IconsEventCard/IconsEventCard.jsx

```diff
@@ -22,7 +22,7 @@ const IconsEventCard = ({ styleForIcon, userId }) => {
 
   return MAPPER.map(
     ({ id, nameControl, styles, todo, visibility }) =>
-      visibility && (
+      visibility !== null && (
```

Comment:
I changed it because if the user id is 0, the check will give false. But now I have a question. What happens if backend doesn't work correctly, what can come from backend. The default value in redux is null. Will redux form replace default value with value from backend if request will not work correctly? I don't know how to check it.